twitter/settings.py

```diff
@@ -209,6 +209,10 @@
 CELERY_TIMEZONE = "UTC"
 # 要不要跑进程的时候也把celery 启动， 这个就是说testing 的时候不用异步，用同步方式执行
 CELERY_TASK_ALWAYS_EAGER = TESTING
+CELERY_QUEUES = (
```

Comment:
key: in order to unblock the other usage of the MQ, we need to have different queues for different purposes